src/main/java/util/csvparsing/CsvFileInterface.java

```diff
@@ -0,0 +1,19 @@
+package util.csvparsing;
+
+/**
+ * Class that defines common methods needed to interface with CSV Files
+ */
+class CsvFileInterface {
```

Comment:
If you have a better name for this class, please let me know lol

Reply:
Steve.java

Seriously though, I wouldn't use Interface in the name if it's a class because it's a little confusing. Maybe `CsvFileHelper` and just instantiate it in the writer/reader instead of extending it